include/rocksdb/configurable.h

```diff
@@ -356,4 +356,8 @@ class Configurable {
   // Configurable option via
   std::vector<RegisteredOptions> options_;
 };
+
+extern void RegisterConfigurableOptions(
```

Comment:
Maybe a comment to explain it will be used by the third party to create their own OptionTypeInfo map for the plugin object configuration options.